src/js/components/BreadcrumbComponent.jsx

```diff
@@ -1,7 +1,12 @@
+var _ = require("underscore");
+var classNames = require("classnames");
 var React = require("react/addons");
 var Link = require("react-router").Link;
 
 var PathUtil = require("../helpers/PathUtil");
+var StyleDimensions = require("../constants/StyleDimensions");
```

Comment:
@pierlo-upitup original `white-space` fix pushed, thanks for that. 

I'll remove the StyleDimensions if you guys don't think they're necessary.